src/main/java/org/ngafid/uploads/process/NioInputFile.java

```diff
@@ -0,0 +1,98 @@
+
+package org.ngafid.uploads.process;
+
+
+import org.apache.parquet.io.InputFile;
+import org.apache.parquet.io.SeekableInputStream;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+
```

Comment:
A comment describing the need for this file would be good (i.e. there is no concrete implementation of `InputFile` available without a bunch of packages).